spec/requests/api/v2/access_request_spec.rb

```diff
@@ -24,6 +24,117 @@
 
   subject { response }
 
+  describe 'GET #index' do
+    let(:access_requests_index_route) do
+      get "/api/v2/access_requests",
+          headers: { 'HTTP_AUTHORIZATION' => credentials }
+    end
+
+    context 'when unauthenticated' do
+      before do
+        access_requests_index_route
+      end
+
+      let(:payload) { { email: 'foo@bar' } }
+
+      it { should have_http_status(:unauthorized) }
+    end
+
+    context 'when unauthorized' do
+      let(:unauthorised_user) { create(:user) }
+      let(:payload) { { email: unauthorised_user.email } }
+      let(:unauthorised_user_route) do
+        get "/api/v2/access_requests",
+            headers: { 'HTTP_AUTHORIZATION' => credentials }
+      end
+
+
+      it "should raise an error" do
+        expect { unauthorised_user_route }.to raise_error Pundit::NotAuthorizedError
+      end
+    end
+
+    context 'when authorised' do
+      let(:first_organisation) { create(:organisation, name: 'First Organisation') }
```

Comment:
Do we need all this setup? Feel like a lot of this can come from the factories instead of being explicitly set.